### ISSUE TITLE
Update alt text for metrics-pie-chart  image on the About Us page to adhere to WCAG

### ISSUE THREAD

**Issue by blulady** [hackforla/website#3404]
### Overview
We need to ensure that alt text for images adhere to Web Content Accessibility Guidelines (WCAG). In this issue, we will change the alt text for the Metrics pie graph image on the [About#Metrics](https://www.hackforla.org/about/#metrics).

### Details
Designers and PMs identified that some alt texts in the Events page should be changed because they are either inaccurate or too long. Ideally, alt text should be descriptive but short and succinct. According to the WCAG page, an accompanying alt text should be short. Additionally, the Functional Accessibility Evaluator (FAE) recommends for alt text to be no more than 100 characters.

### Action Items
- [x] In the file `_includes/about-page/about-card-metrics.html`, add the specified alt code to the following line (the code is intentionally `alt=""` (the tag is supposed to be empty):

From
```
Piechart of Metrics Breakdown
```

To
```
alt=""
```

- [ ] **For merge team**: Once the pull request associated with this issue is approved and merged, please update and edit epic #3361 by
  - [ ] Checking off the dependency for this issue
  - [ ] If all dependencies are checked off, please move issue #3361 to the New Issue Approval column and remove the Dependency label

### Resources/Instructions
- This issue is part of epic #3361 
- [About Us page](https://www.hackforla.org/about/)
- [Alt text decision Tree ](https://www.w3.org/WAI/tutorials/images/decision-tree/)
- [FAE rule 4: alt text should be less than 100 characters](https://fae.disability.illinois.edu/rulesets/IMAGE_4_EN/)

#### File and code links you will need to work on this issue
- Directory to find the page once you have it in your IDE: `_includes/about-page/about-card-platform.html`
- Click on the link below to see the source file code:
https://github.com/hackforla/website/blob/a83dd650b664618ff2352b2fb93e8ae4acfb402b/_includes/about-page/about-card-platform.html#L77

#### Information about WCAG
- [WCAG](https://www.w3.org/WAI/standards-guidelines/wcag/)
- [Web Accessibility Tutorials: Decorative Images](https://www.w3.org/WAI/tutorials/images/decorative/)
- [WCAG section on alt text](https://www.w3.org/WAI/WCAG21/Techniques/html/H37.html)

**Comment by JohnJBarrett22** [comment]
i. Availability: (When are you available to work on the issue/answer questions other programmers might have about your issue?) Almost anything aside from shifting doc appointments for my Wife.
ii. ETA: (When do you expect this issue to be completed?) This issue is completed, pending review and merge.

**Comment by JohnJBarrett22** [comment]
Progress: This was just a good second issue project. It is complete and pending review and merge. Once that is done, I will move to the next tier of issues."
Blockers: N/A
Availability: 30ish minutes total, 2 for the issue itself and the rest making sure I did git commands correctly.
ETA: Issue is done.
Pictures (optional): N/A
